config/routes.rb

```diff
@@ -1,4 +1,8 @@
 Rails.application.routes.draw do
+  namespace :account do
```

Comment:
why one more time namespace account?
we already have it

Reply:
Fixed, thanks.